client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/pages/motion-form/components/motion-form/motion-form.component.ts

```diff
@@ -207,6 +211,11 @@ export class MotionFormComponent extends BaseMeetingComponent implements OnInit
     public saveMotion(event?: any): () => Promise<void> {
         return async () => {
             const update = event || this.temporaryMotion;
+            for (const key in update) {
+                if (update[key] === null || update[key].length === 0) {
+                    delete update[key];
+                }
```

Comment:
Probably not a good idea: null's the value we use to signify that a value is to be deleted, removing all keys with that value is going to remove that functionality. Also you can't guarantee that every key is an array.

What I think you really want to do, is removing certain keys depending on whether the user is even allowed to set them according to his permissions

Reply:
for the motion.create it is enough to remove every null/empty array from the payload. The permissions do not have to be verified, given that a user can only send null/[] for the fields they are not allowed to change

For the motion.update it is wrong to remove empty arrays or null values